test/acceptance/test/help_tests.go

```diff
@@ -18,73 +18,73 @@ var _ = XDescribe("WEGO Help Tests", func() {
 
 	BeforeEach(func() {
 
-		By("Given I have a wego binary installed on my local machine", func() {
+		By("Given I have a gitops binary installed on my local machine", func() {
 			Expect(FileExists(WEGO_BIN_PATH)).To(BeTrue())
 		})
 	})
 
-	It("Verify that wego displays error message when provided with the wrong flag", func() {
+	It("Verify that gitops displays error message when provided with the wrong flag", func() {
 
-		By("When I run 'wego foo'", func() {
+		By("When I run 'gitops foo'", func() {
 			command := exec.Command(WEGO_BIN_PATH, "foo")
 			sessionOutput, err = gexec.Start(command, GinkgoWriter, GinkgoWriter)
 			Expect(err).ShouldNot(HaveOccurred())
 		})
 
-		By("Then I should see wego error message", func() {
-			Eventually(sessionOutput.Err).Should(gbytes.Say("Error: unknown command \"foo\" for \"wego\""))
-			Eventually(sessionOutput.Err).Should(gbytes.Say("Run 'wego --help' for usage."))
+		By("Then I should see gitops error message", func() {
+			Eventually(sessionOutput.Err).Should(gbytes.Say("Error: unknown command \"foo\" for \"gitops\""))
+			Eventually(sessionOutput.Err).Should(gbytes.Say("Run 'gitops --help' for usage."))
```

Comment:
+1 for that "text catalog" thing we talked about a while back for matching up output with a known variable.

Reply:
Seriously!